src/main/resources/templates/product/show_products.ftl

```diff
@@ -86,9 +103,16 @@
                 <a style='display:inline; width: 50px' href="/products/update/${product.id}">
                 <button class="btn btn-light" type="submit">Edit</button>
                 </a>
-                <form style='display:inline; width: 50px' action="/products/delete/${product.id}" method="post">
-                    <button class="btn btn-dark" type="submit">Delete</button>
-                </form>
+                <#if product.active==true>
+                    <form style='display:inline; width: 50px' action="/products/delete/${product.id}" method="post">
+                        <button class="btn btn-dark" type="submit">Delete</button>
```

Comment:
`Archivate` - better name in this case

Reply:
Better "Archive", there is no word "Archivate" in English :D